tune_sklearn/_trainable.py

```diff
@@ -42,13 +42,12 @@ def _setup(self, config):
         self.groups = config.pop("groups")
         self.fit_params = config.pop("fit_params")
         self.scoring = config.pop("scoring")
-        self.early_stopping = config.pop("early_stopping")
         self.max_iters = config.pop("max_iters")
         self.cv = config.pop("cv")
         self.return_train_score = config.pop("return_train_score")
         self.estimator_config = config
 
-        if self.early_stopping:
+        if self.early_stopping is not None:
```

Comment:
Should we also have the same conditional in the _train method (line 79) `if self.early_stopping is not None:`. Just so it's more clear and matches with this part.

Reply:
Actually there's a few places where both versions of this check is used in all the other files (tune_basesearch.py, tune_gridsearch.py, tune_search.py). Doesn't `if self.early_stopping:` do the same thing as `if self.early_stopping is not None:`? Either way it would be nice to have it consistent.

Reply:
Yep, missed some spots and will fix